pinot-common/src/main/java/org/apache/pinot/common/utils/config/QueryOptionsUtils.java

```diff
@@ -173,6 +173,15 @@ public static Integer getMinServerGroupTrimSize(Map<String, String> queryOptions
     return minServerGroupTrimSizeString != null ? Integer.parseInt(minServerGroupTrimSizeString) : null;
   }
 
+  @Nullable
+  public static Integer getMinBrokerGroupTrimSize(Map<String, String> queryOptions) {
+    if (queryOptions == null) {
+      return null;
+    }
```

Comment:
Moved this check into (Streaming/Broker)ReduceService. This check is needed because unlike servers (where queryOption cannot be null because of timeout,etc), it can be null for brokers.